content/en/docs/kubernetes/collector/components.md

```diff
@@ -13,13 +13,148 @@ enhancing it.
 
 Important Components:
 
+- [Kubernetes Attributes Processor](#kubernetes-attributes-processor) - used to
+  add Kubernetes metadata to incoming telemetry.
 - [Filelog Receiver](#filelog-receiver) - used to collect Kubernetes logs and
   application logs written to stdout/stderr.
 
 For application traces, metrics, or logs, we recommend the
 [OTLP receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver),
 but any receiver that fits your data is appropriate.
 
+## Kubernetes Attributes Processor
+
+The Kubernetes Attributes Processor automatically discovers Kubernetes resources
+(pods), extracts their metadata, and adds the extracted metadata to the relevant
+spans, metrics and logs as resource attributes.
+
+**It is one of the most important components for a collector running in
+Kubernetes; any collector receiving application data should use it.** This is
+the processor that adds Kubernetes context to your application traces, metrics,
+and logs. It enables you to correlate your application's traces, metrics, and
+logs with other Kubernetes telemetry such as pod metrics and events.
+
+The Kubernetes Attributes Processor uses the Kubernetes API to discover all pods
+running in a cluster, keeps a record of their IP addresses, pod UIDs and
+interesting metadata. By default data passing through the processor is
+associated to a pod via the incoming request's IP address, but different rules
+can be configured. Since the processor uses the Kubernetes API it requires
+special permissions. If you're using the
+[OpenTelemetry Collector Helm chart](../../helm/collector/) you can use the
+[`kubernetesAttributes` preset](../../helm/collector/#kubernetes-attributes-preset)
+to get started.
```

Comment:
```suggestion
The Kubernetes Attributes Processor uses the Kubernetes API to discover all pods
running in a cluster and keeps a record of their IP addresses, pod UIDs, and
interesting metadata. By default, data passing through the processor is
associated to a pod via the incoming request's IP address, but different rules
can be configured. Since the processor uses the Kubernetes API, it requires
special permissions (see example below). If you're using the
[OpenTelemetry Collector Helm chart](../../helm/collector/) you can use the
[`kubernetesAttributes` preset](../../helm/collector/#kubernetes-attributes-preset)
to get started.
```